src/firefly/js/visualize/XYPlotCntlr.js

```diff
@@ -283,6 +320,7 @@ function fetchPlotData(dispatch, activeTableServerRequest, xyPlotParams) {
             console.error(`Failed to fetch XY plot data: ${reason}`);
         }
     );
+
 }
 
 
```

Comment:
1. getInitState() is not needed since it returns {}.  export function reducer(state={}, action={}) should be fine.
2. console.error should change to logError instead.
3. Is any reason that some places the function expressions are used and other places function declarations are used?